databricks_blueprints/delete_file_from_dbfs.py

```diff
@@ -0,0 +1,65 @@
+"""
+
+"""
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+try:
+    import helpers
+    import errors
+except BaseException:
+    from . import helpers
+    from . import errors
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--instance-id', dest='instance_id', required=True)
+    parser.add_argument('--source-file-name', dest='source_file_name', required=True)
+    parser.add_argument('--source-folder-name', dest='source_folder_name', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def delete_file_from_dbfs(client, file_path_and_name):
+    delete_endpoint = "/dbfs/delete"
+    data = {
+        'path': file_path_and_name
+    }
+    print(f"Start delete for {file_path_and_name}")
+    delete_response = client.post(delete_endpoint, data=data)
+    if delete_response.status_code == requests.codes.ok:
+        print(f"DBFS File: {file_path_and_name} delete function completed successfully")
+    elif delete_response.status_code == 503: # Too many files left to delete
+        delete_json = delete_response.json()
+        error_code = delete_json['error_code']
+        error_message = delete_json['message']
+        if error_code == 'PARTIAL_DELETE':
+            print(error_message)
+            sys.exit(errors.EXIT_CODE_DBFS_FILE_OVERFLOW)
+        else: # This is a regular Service unavailable error
+            print("File delete error: Service currently unavailable")
+            sys.exit(errors.EXIT_CODE_DBFS_SERVICE_UNAVAILABLE)
+    else:
+        print(f"File delete failed. Response: {delete_response.text}")
+        sys.exit(errors.EXIT_CODE_UNKNOWN_ERROR)
+        
+
+def main():
+    args = get_args()
+    access_token = args.access_token
+    instance_id = args.instance_id
+    source_file_name = args.source_file_name
+    source_folder_name = args.source_folder_name
```

Comment:
Needs to use shipyard_utils cleaning

Reply:
@shamwayne this still needs to use shipyard_utils for cleaning the folder name.

If I provide the folder name as `/FileStore`, `/FileStore/` or `FileStore/`, it should work in all cases. Right now, if I don't provide the folder with slashes before and after, it returns  Response: {"error_code":"INVALID_PARAMETER_VALUE","message":"Path must be absolute: FileStore/10.csv"}